.editorconfig

```diff
@@ -1,295 +1,45 @@
-root = true
+# WARNING: Only edit this file in the Lombiq .NET Analyzers repository's "Lombiq.Analyzers" folder. A copy of this file
```

Comment:
I think we should be able to override Lombiq's project using this file. Lombiq may have perfect settings buy default, but if someone disagree, we should be able to override it here. This maybe a better way so that other people can use the Lombiq's project as their base and tweak what they don't like

Reply:
I rolled back the changes and started from scratch (in terms of editor settings).